lib/resources/storage/web_storage.dart

```diff
@@ -10,4 +14,28 @@ class StorageProvider {
       return null;
     }
   }
+
+  static Future<bool> uploadImage(
+      BuildContext context, Uint8List file, String ref) async {
+    try {
+      final StorageReference storageReference = storage().ref('').child(ref);
+
+      final UploadTaskSnapshot uploadTaskSnapshot =
+          await storageReference.put(file).future;
```

Comment:
Isn't `put` deprecated in favour of `putFile`?

Reply:
From my version it is not. The `putFile` method does not exist